app/route_rangers_api/urls.py

```diff
@@ -1,11 +1,15 @@
 from django.urls import path
-
 from . import views
-from route_rangers_api.views import MarkersMapView
+from views import MarkersMapView
 
 app_name = "app"
 
 urlpatterns = [
     path("", views.index, name="index"),
-    path("map/", MarkersMapView.as_view()),
+    path("home/", views.home, name="home"),
```

Comment:
I think index should be replaced with the content from home as index is just a placeholder for now, i.e. rename views.home as views.index and delete the home path

Reply:
fixed